components/Dashboard/Hero.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div class="grid grid-cols-1 md:grid-cols-2 items-center gap-5 my-5 lg:my-20">
+    <h1>{{ t('dashboard.title') }}</h1>
+    <p class="text-xl">{{ t('dashboard.description') }}</p>
```

Comment:
This needs to be an 4xl instead of xl.
Please check with the design provided in penbot. TO figure out the text size just click on the text and it will show on the panel on the rigth.

Reply:
resolved in [d55f31b](https://github.com/jordanopensource/nuha-web/pull/89/commits/d55f31b6c6e7becb8bfd46c38269450a6c0875b3)